apps/admin/frontend/src/components/election_manager.tsx

```diff
@@ -53,7 +53,13 @@ export function ElectionManager(): JSX.Element {
     return (
       <UnlockMachineScreen
         auth={auth}
-        checkPin={(pin) => checkPinMutation.mutate({ pin })}
+        checkPin={async (pin) => {
+          try {
+            await checkPinMutation.mutateAsync({ pin });
+          } catch {
+            // Handled by default query client error handling
+          }
+        }}
```

Comment:
Not sure what the best pattern here is - using this `mutateAsync` or accepting a callback in additional to the `pin` that you use as `mutate`'s `onSuccess` callback.